core/src/main/java/org/apache/gravitino/authorization/AuthorizationUtils.java

```diff
@@ -190,6 +207,10 @@ public static boolean needApplyAllAuthorizationPlugin(SecurableObject securableO
     return false;
   }
 
+  private static boolean needApplyAuthorizationPluginSetOwner(MetadataObject.Type type) {
+    return type == MetadataObject.Type.METALAKE;
+  }
+
   private static boolean supportsSingleAuthorizationPlugin(MetadataObject.Type type) {
```

Comment:
What does `Single` mean in the `supportsSingleAuthorizationPlugin()` function?
Can we have a better name?

Reply:
I changed to `supportsPushdownAuthorizationPlugin`.